### ISSUE TITLE
Experiment 3

### ISSUE THREAD

**Issue by duggar** [tschm/cs#1]
Hi Dr. Schmelzer,
First of all thanks for sharing this work, and I am following the code line by line where I found a potential issue(which might be my lack of understanding), 
so we have this right now:

portfolio = Portfolio(prices=prices, position=prices.apply(f, slow=slow.value, fast=fast.value, vola=vola.value, clip=winsor.value))

where the position passed to Portfolio Class varies quite violently, I'd rather have it like this:

position=prices.apply(f, slow=slow.value, fast=fast.value, vola=vola.value, clip=winsor.value)

![image](https://user-images.githubusercontent.com/37486846/184679281-6efb7886-4377-425a-9d0f-bae7329b9fb5.png)
### plus if would want to avoid leverage 
position = position.apply(lambda x: x/x.abs().sum(), axis=1)
and then pass this to
portfolio = Portfolio(prices=prices, position=position)

Basically not just experiment 3 but in all the schemes. Do let me know, I might be terribly wrong.

**Comment by duggar** [comment]
This might be a distortion from what you intended as the net position sums to 1

**Comment by tschm** [comment]
@duggar Thank you for your question. I assume you come from equity world where a position is typically a fraction of some capital C. The sum of those weights typically has to sum to 0 (cash-neutral) or 1 (fully invested). 

Here, I am dealing with futures. Futures have built-in leverage, e.g. you buy a contract on an underlying index. This contract is cited in points. Make sure you understand https://www.barchart.com/futures/contract-specifications/indices

The S&P mini is currently at 4300 points or 4300 x 50$ if you prefer a cash position. This cash position is relevant but as an investor you only put down the margin requirement. At the end of the day the siyouze of the cash position is important but even more relevant are the daily fluctuations of this quantity, hence the position I am interested is the cash position x daily volatility. This quantity should be on roughly the same scale for all futures you are holding and that's the decisive variable in my optimization attempts.

Once you know the risk position one can step back to cash positions and from there to number of contracts (lots).

I hope this answer helps you.

**Comment by duggar** [comment]
Thanks for the clarification, really helpful.
